sqlmesh/core/context.py

```diff
@@ -900,6 +900,7 @@ def evaluate(
     def format(
         self,
         transpile: t.Optional[str] = None,
+        no_rewrite_casts: t.Optional[bool] = None,
```

Comment:
This API is now a bit inverted internally.

Would `rewrite_casts: bool = True` make more sense? Then the CLI can set it to `False` if `--no-rewrite-casts` was set.

Reply:
Done